openapi.yaml

```diff
@@ -5545,6 +5545,83 @@ paths:
       - lang: CLI
         source: >
           linode-cli linodes transfer-view 123
+  /linode/instances/{linodeId}/transfer/{year}/{month}:
+    parameters:
+    - name: linodeId
+      in: path
+      description: ID of the Linode to look up.
+      required: true
+      schema:
+        type: integer
+    - name: year
+      in: path
+      description: Numeric value representing the year to look up.
+      required: true
+      schema:
+        type: integer
+        minimum: 2000
+        maximum: 2037
```

Comment:
In 17 years, we will have to worry about this 😆

Reply:
lol!